packages/admin_view/src/components/LexiconTable.tsx

```diff
@@ -0,0 +1,118 @@
+import { GetAllLexEntriesQuery, useGetAllLexEntriesQuery } from '../graphql/lexicon/lexicon.ts';
+import { DataGrid, GridColDef,DataGridProps } from '@mui/x-data-grid';
+import React, { useState } from 'react';
+import { Box, Stack, Button } from '@mui/material';
+import { alpha, createTheme, styled } from '@mui/material/styles';
+
+const theme = createTheme({
+  palette:{
+    primary:{
+      light: '#757ce8', 
+      main: '#3f50b5', 
+      dark: '#002884',   
+    }
+  }
+})
+const StyledDataGrid = styled(DataGrid)<DataGridProps>(({ theme }) => ({
+  '&.MuiDataGrid-root': {
+    backgroundColor: alpha(theme.palette.primary.light, 0.3),
+  },
+  //don't know why this is not applied
+  '.MuiDataGrid-columnHeaders': {
+    backgroundColor: alpha(theme.palette.primary.dark, .8),
+    fontWeight: 'bold',
+
+  },
+  '.MuiDataGrid-cell': {
+    backgroundColor: alpha(theme.palette.primary.light, 0.5),
+  },
+}));
+
+
+const tableColumns: GridColDef<GetAllLexEntriesQuery['lexiconAllEntries'][number]>[] = [
+  {
+    field: 'primary',
+    headerName: 'Primary',
+    width: 200,
+    headerAlign: 'center',
+    headerClassName: 'boldHeader'
```

Comment:
This classname is never defined so I don't think this does anything

Reply:
oh yeah I was trying to make the headers bold, but it is not working so I will remove it, definitely not a priority right now